archivebox/parsers/pocket_api.py

```diff
@@ -0,0 +1,115 @@
+__package__ = 'archivebox.parsers'
+
+
+import re
+
+from typing import IO, Iterable, Optional
+from datetime import datetime
+from configparser import ConfigParser
+
+from pathlib import Path
+from pocket import Pocket
+import requests
+
+from ..index.schema import Link
+from ..util import (
+    enforce_types,
+)
+from ..config import (
+    OUTPUT_DIR
+)
+
+_COUNT_PER_PAGE = 500
+_SINCE_FILENAME = Path(OUTPUT_DIR) / 'pocket_api.db'
```

Comment:
I think it makes sense to put it in the SOURCES_DIR subfolder, not the root dir.

Reply:
Done.